include/zephyr/devicetree.h

```diff
@@ -2203,6 +2203,14 @@
  */
 #define DT_REG_ADDR(node_id) DT_REG_ADDR_BY_IDX(node_id, 0)
 
+/**
+ * @brief 64-bit version of a DT_REG_ADDR macro.
+ *        Use it in cases when 64-bit address value may appear in Device Tree
+ * @param node_id node identifier
+ * @return node's register block address
+ */
+#define DT_REG_ADDR_U64(node_id) UINT64_C(DT_REG_ADDR(node_id))
```

Comment:
Please split the devicetree API changes into their own commit(s), separate from the commit modifying your platform.

Reply:
Ok, makes sense